tests/v2/waku_archive/test_waku_archive.nim

```diff
@@ -367,8 +358,18 @@ procSuite "Waku Archive - find messages":
     var pages = newSeq[seq[WakuMessage]](3)
     var cursors = newSeq[Option[ArchiveCursor]](3)
 
+    let archiveA = block:
+      let
+        driver = newTestArchiveDriver()
+        archive = newTestWakuArchive(driver)
+
+      for msg in msgListA:
+        require (await driver.put(DefaultPubsubTopic, msg, computeDigest(msg), msg.timestamp)).isOk()
+
+      archive
```

Comment:
Perhaps if you use `waitFor` rather than `await` this wouldn't need to move into the async block?